pkg/controller/performanceprofile/performanceprofile_controller.go

```diff
@@ -371,39 +352,6 @@ func (r *ReconcilePerformanceProfile) applyComponents(profile *performancev1alph
 		return nil, err
 	}
 
-	updated := (mcMutated != nil ||
```

Comment:
actually, it might be worth keeping this `updated` boolean. It indicates no changes are going ot occur and that we can break out immediately. This would help with knowing when to post events like "all resources have been created" only once as well.

Reply:
oh, good catch, thanks. I was a bit too enthusiastic with deleting code here ;) Fixed.